### PR TITLE
Fix security database schema

### PR DESCRIPTION
**Pull request by tshemsedinov**

### CODE REVIEW

File: schemas/security/SystemLog.schema [hunk 1]
```diff
@@ -2,5 +2,6 @@
   User: { category: 'SystemUser', required: true },
   Category: { domain: 'Id', required: true },
   Timestamp: { domain: 'Timestamp', requires: true },
-  Operation: { domain: 'UserAccess', requires: true }
+  Operation: { domain: 'UserAccess', requires: true },
```
Comment (lundibundi) [inline schemas/security/SystemLog.schema]: Shouldn't it be `required`? (here and above)

Comment (tshemsedinov) [reply schemas/security/SystemLog.schema]: We will log certain operations, so I think each log record should refer to one

Comment (tshemsedinov) [reply schemas/security/SystemLog.schema]: You are right, I misspelled that

**Comment by tshemsedinov** [comment]
Landed in ca1ddb0

**Pull request closed without merging**